woocommerce/woocommerce-gutenberg-products-block
# Add Express Checkout block for Cart i2

Adds Express Checkout Block (Copied as is from Checkout block with minor changes).

This PR adds Express Checkout Block to cart. It's copied from Express Checkout block for Checkout block with minor changes, I see a chance to reuse much of the code here.

Part of #4709, based on #4780

Testing Steps
Insert Cart i2.
You should see the express placeholder or express preview depending if you have them enabled or not.
![image](https://user-images.githubusercontent.com/6165348/134033970-f5eb3d8d-1d8e-48b5-9029-f54cae59ca7d.png)

![image](https://user-images.githubusercontent.com/6165348/134034010-10d2c4bb-be94-4b7d-9460-bf5f5176587f.png)
